src/rosdep2/lookup.py

```diff
@@ -339,6 +339,8 @@ def create_from_rospkg(rospack=None, rosstack=None,
           instance used to crawl ROS stacks.
         :param sources_loader: (optional) Override SourcesLoader used
             for managing sources.list data sources.
+        :param dependency_types: (optional) List of dependency types.
+            Allowed: "build", "buildtool", "run", "test"
```

Comment:
```suggestion
            Allowed: "build", "buildtool", "run", "test", "exec", "doc"
```
It also supports exec and doc dependencies

Reply:
Should these docs also link to the specification for dependency types either in catkin_pkg or the ROS REPs defining the package.xml format?

Reply:
Good point. The `doc_depend` seems to be introduced in format 2 ( [REP127](https://ros.org/reps/rep-0127.html), [schema files](https://github.com/ros-infrastructure/rep/blob/master/xsd/package_format1.xsd) ), so maybe it is better not to consider it.